apps/modernization-ui/src/apps/search/investigation/CriteriaSearchFields.tsx

```diff
@@ -7,11 +7,28 @@ import {
     processingStatusOptions
 } from './InvestigationFormTypes';
 import { SingleSelect, MultiSelect } from 'design-system/select';
-import { ConceptAutocomplete } from 'options/autocompete/ConceptAutocomplete';
+import { ConceptOptionsService } from 'generated';
+import { useEffect, useState } from 'react';
+import { Selectable } from 'options';
 
-const CriteriaSearchFields = () => {
+type Props = {
+    outbreakCodeSetNm?: string;
+};
+
+const CriteriaSearchFields = ({ outbreakCodeSetNm = 'OUTBREAK_NM' }: Props) => {
```

Comment:
Question: Do we need to pass in this as a prop? When will it be a different value?

Reply:
Just changed after adams suggestion of creating a lone component to handle the concept multi selects that could be used elsewhere as well